components/worktemplates/components/customize.tsx

```diff
@@ -0,0 +1,119 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import styled from 'styled-components';
+import {useIntl} from 'react-intl';
+
+import PublicPrivateSelector from 'components/widgets/public-private-selector/public-private-selector';
+import {Visibility} from '@mattermost/types/worktemplates';
```

Comment:
nit: keep imports grouped by context, in this case the 'components' one together and the '@mattermost' in a separate lines